gammapy/irf/psf/tests/test_parametric.py

```diff
@@ -88,6 +88,17 @@ def test_psf_cta_1dc():
     assert_allclose(radius, 0.052841 * u.deg, atol=1e-4)
 
 
+@requires_data()
+def test_get_sigmas_and_norms():
+    filename = "$GAMMAPY_DATA/cta-caldb/Prod5-South-20deg-AverageAz-14MSTs37SSTs.180000s-v0.1.fits.gz"
+
+    psf_irf = EnergyDependentMultiGaussPSF.read(filename, hdu="POINT SPREAD FUNCTION")
+
+    value = psf_irf.evaluate(
+        energy_true=0.05 * u.TeV, rad=0.03 * u.deg, offset=4.5 * u.deg
+    )
+    assert_allclose(value, 0 * u.Unit("deg-2"))
```

Comment:
Maybe choose a value that is non-zero here?

Reply:
Ok.